tools/loader/ImageClassifier.cpp

```diff
@@ -241,8 +246,15 @@ int main(int argc, char **argv) {
     for (unsigned i = 0; i < inputImageFilenames.size(); i++) {
       Tensor slice = H.extractSlice(i);
       auto SH = slice.getHandle<>();
+      size_t result;
```

Comment:
Could you simply do `size_t result = SH.minMaxArg().second - labelOffset;`? And change the comment to something like:
```
        // Some models are trained with more classes. E.g. imagenet models
        // exported from TensorFlow have 1 extra "neutral" class.
```

Reply:
ok